acceptance/fixtures/files/retry_script.txt

```diff
@@ -0,0 +1,14 @@
+
+# This script which will fail the first `RETRY_LIMIT` times when run and then
+# will exit successfully on later runs. It will store counter state in a file
+# in the provided `BASEDIR` directory.
+set -x
+
+BASEDIR=${1}
+RETRY_LIMIT=${2}
+
+current=`cat ${BASEDIR}/value.txt || echo '0'`
```

Comment:
This is probably not too important right now, but I believe that this will fail on powershell windows boxes (windows without cygwin).